app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+# Controller for static pages
+class StaticPagesController < ApplicationController
+  def new() end
```

Comment:
is the `()` necessary?

Reply:
Are you actually using this action or Controller? 
Either way can write it like `def new; end`  [OPTIONAL]

Reply:
As per rubocop for empty methods

Reply:
It's to map the home page, to a static page. `def new() end` is perfectly valid.

Reply:
Okay, I was simply asking. As you know, I do not have time to go through every single piece of code, and I never required you to make this change. never said it was not valid

Reply:
But you are absolutely correct, and I apologise